modules/n1ql/pages/n1ql-language-reference/createfunction.adoc

```diff
@@ -91,7 +91,7 @@ parameter::
 [Optional] The name of a function argument.
 If you specify named parameters for the function, then you must call the function with exactly the same number of arguments at execution time.
 If you specify no parameters, then you must call the function with no arguments.
-To create a variadic function, that is, a function which you can call with any number of arguments or none, specify `...` as the only parameter.
+To create a variadic function, that is, a function which you can call with any number of arguments or none, specify `\...` as the only parameter.
```

Comment:
Quick question: I don't understand this syntax - is that escaping all 3 dots with a single character? And is the escape relevant in N1ql or at Asciidoc level?

Reply:
The syntax is meant to be three dots in N1QL. If you don't include the backslash, Asciidoctor transforms three dots into an ellipsis. The backslash makes sure that the three dots appear as three dots in the output.

Refer to [character replacement substitutions](https://docs.asciidoctor.org/asciidoc/latest/subs/replacements/) and [escaping text substitutions](https://docs.asciidoctor.org/asciidoc/latest/subs/prevent/#escape-with-backslashes).